README.md

```diff
@@ -1,6 +1,16 @@
 meta-rte
 ========
 
+# Meta-RTE Yocto BSP documentation
+
+* [BSP layers description](docs/layer-description.md)
+
+* [Development process description](docs/development.md)
+
+* [Release process description](docs/release.md)
+
+* [Instruction of setting the MAC address in U-Boot](docs/rte-mac-setup.md)
```

Comment:
@PLangowski let's add here also a link for cpu notes.

Reply:
done 4d38fd7f2dd3fab263b08ce6fce853abf4b02870